source/_components/tts.markdown

```diff
@@ -33,6 +33,8 @@ The following optional parameters can be used with any platform. However, the TT
 | `cache` | True    | Allow TTS to cache voice file to local storage. |
 | `cache_dir`  | tts      | Folder name or path to a folder for caching files. |
 | `time_memory`     | 300     | Time to hold the voice data inside memory for fast play on a media player. Minimum is 60 s and the maximum 57600 s (16 hours). |
+| `base_url` | '' | A base URL to use *instead* of the one set in the [http component](/components/http/). It is used as-is by the `tts` component. In particular, you need to include the protocol scheme `http://` or `https://` and the correct port number. They will not be automatically added for you. An empty string (the default) means that `tts` will use the one set in the [http component](/components/http/). |
```

Comment:
Rather than `''` maybe either leave it blank or specify that it's the one from HTTP?

That said, this would also be a good time to update the configuration block to use the [one in the standards](https://www.home-assistant.io/developers/documentation/standards/)

Reply:
Yes, given the code changes requested to [code PR](https://github.com/home-assistant/home-assistant/pull/16478), modifying this column is part of the plan.

I'll take a look at the configuration block template. Making the change should be simple enough.